app/src/main/java/com/ccc/nameapp/data/source/remote/api/middleware/RxErrorHandlingCallAdapterFactory.kt

```diff
@@ -65,33 +70,33 @@ class RxErrorHandlingCallAdapterFactory private constructor() : CallAdapter.Fact
                     isMaybe -> "Maybe"
                     else -> "Observable"
                 }
-                val errorMessage = "$name return type must be parameterized as $name <Foo> or $name <? extends Foo>"
+                val errorMessage = "$name return type must be parameterized"
                 throw IllegalStateException(errorMessage)
             }
 
             return when {
-                isFlowable -> (wrapped?.adapt(call) as Flowable<*>).onErrorResumeNext { throwable: Throwable ->
-                    Flowable.error(
+                isFlowable -> (wrapped?.adapt(call) as Flowable<*>).onErrorResumeNext {
+                        throwable: Throwable -> Flowable.error(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/08a2d406ec58d7b52d91c64120c3c4101a7e87e2/app/src/main/java/com/ccc/nameapp/data/source/remote/api/middleware/RxErrorHandlingCallAdapterFactory.kt#L79"></span>Unexpected indentation (24) (should be 20)</td>
    </tr>
  </tbody>
</table>